Devscord.DiscordFramework.UnitTests/Commands/RunnerOfIBotCommandMethodsTests/Tests/MechanismToResolveConflictSituationsTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Devscord.DiscordFramework.UnitTests.Commands.RunnerOfIBotCommandMethodsTests.Controllers;
+using Devscord.DiscordFramework.UnitTests.Commands.RunnerOfIBotCommandMethodsTests.BotCommands;
+using Devscord.DiscordFramework.Framework.Commands.Parsing;
+using NUnit.Framework;
+
+namespace Devscord.DiscordFramework.UnitTests.Commands.RunnerOfIBotCommandMethodsTests.Tests
+{
+    // "Conflict situations" means situations in which a given command matches default and custom version of a bot command at the same time. During these situations, "mechanism to resolve conflict situations" is runned. 
```

Comment:
jeśli musisz dawać opis o tym rozmiarze to znaczy że coś tu jest nie tak

Reply:
niezależnie w którym miejscu w kodzie -> jeśli jest potrzeba dawania komentarza to coś jest nie tak
a jeśli jest potrzeba dawania komentarza dłuższego niż jedno zdanie, to coś jest bardzo mocno nie tak